### PR TITLE
#6460: upgrade amount of users

### PR DESCRIPTION
**Pull request by Sighne**
Changes the amount of users to a million and adds the Discovery Magazine logo on static page (https://github.com/HabitRPG/habitrpg/pull/6460).

### CODE REVIEW

**Comment by Alys** [comment]
There's a second instance of "900,000" that needs to be replaced in front.jade around line 582:

```
      #footercall
        .container-fluid
          .row
              h3= env.t('joinOthers', {userCount:'900,000'})
```

It's also in website/views/social/landing-page.jade :

```
      li=env.t('landingCopy3', {userCount:'900,000'})
```

It would be great if this number could be inserted in one place in a variable.

**Comment by Sighne** [comment]
Thank you, I'll take a look at it.

**Comment by Sighne** [comment]
Changes have been made.
I've also added a variable in routes/pages.js that is passed to the templates in static and social.

File: website/src/routes/pages.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ var _ = require('lodash');
 var locals = require('../middlewares/locals');
 var i18n = require('../libs/i18n');
 
+var userCount = '1,000,000';
```
Comment (crookedneighbor) [inline website/src/routes/pages.js]: Let's make this a constant.

``` js
const TOTAL_USER_COUNT = '1,000,000';
```

**Comment by Sighne** [comment]
Good idea! I changed it.

File: website/src/routes/pages.js (hunk 1 above)
Comment (paglias) [inline website/src/routes/pages.js]: hmm @crookedneighbor is const supported in the api v2 environment? it's still on node 0.10

Comment (paglias) [inline website/src/routes/pages.js]: I forgot we were using babel here too, ignore the comment ^

**Comment by SabreCat** [comment]
Looking good! This will count toward a contributor tier. @Sighne, what's your Habitica User ID? (See https://habitica.com/#/options/settings/api or Settings in the mobile apps.)

**Comment by Sighne** [comment]
Thank you!
Here is my ID: b8c9e0c3-dfe7-4025-b141-d7289ed8e935

**Comment by SabreCat** [comment]
Tier awarded! Note that while your first tier is easy to get, subsequent tiers require increasing amounts of effort from one to the next. So level 2 will require multiple PRs, or a larger PR, to attain. In any case, keep helping out and we'll express our thanks accordingly!

**Pull request merged**